packages/init/src/index.ts

```diff
@@ -41,46 +49,39 @@ async function init() {
   const frameworkResult = await detectFramework(cwd);
 
   // select the repository used with the project.
-  const { existing, name } = await maybeExistingRepo(
+  const { existing, repository } = await maybeExistingRepo(
     config.cookies,
     cwd,
     config.base
   );
 
   if (!existing) {
-    await createRepository(name, frameworkResult.value, config);
+    await createRepository(repository, frameworkResult.value, config);
+  }
+
+  // If we get the info from the profile we want to identify all the previous events sent or continue in anonymous mode
+  if (user.profile) {
+    Tracker.get().identifyUser(user.profile.userId);
```

Comment:
- Shouldn't we use the shortID here ?
- I think this should be done right after getting the user.
- I personally don't see the point to identify the user for one event (start) :x, I would rather have had only this event when the user logs in and put the login at the first position in the steps.
- I think profile should never be undefined, what I mean is if you're identified, the profile call should not fail. Maybe the profile call could be what we use to verify that the user is logged in.

Reply:
Furthermore after looking at the code, i'm more convince about point 3.
The login being the first step and a mandatory step, if the user doesn't login it means he didn't start the script so I would definitely
- let the user login
- create the tracker with profile data
- launch the start event right after the initialisation.